Added Signal Hub specs (#812)
italia/eid-wallet-it-docs: docs/en/credential-revocation.rst

The (Q)EAA remains **Suspended** until it is restored to the **Issued** or **Valid** state (**(Q)EAA UNSUSP**) depending on the previous state, i.e.
the conditions leading to its suspension are resolved, or it changes in **Revoked**, **Expired** or it is deleted. The suspension of a (Q)EAA MAY be:

>   * Use case driven, based on the validity status of the attributes contained in the (Q)EAA. In this case, an Authentic Source MUST notify through Signal Hub the Credential Issuer of any changes in the state of the attributes attested by the (Q)EAA.

Reviewer: ```suggestion
  * Use case driven, based on the validity status of the attributes contained in the (Q)EAA. In this case, an Authentic Source MUST use Signal Hub to notify the Credential Issuer of any changes in the state of the attributes attested by the (Q)EAA.
```

Authentic Sources manage attributes separately from Digital Credentials, which verify authenticity like physical documents. Losing a physical document doesn't mean losing the privileges it represents; it just means the User can't prove them. However, if a User loses privileges due to a serious infraction, the Authentic Source will revoke the related attributes. In such cases, when a User's attributes are updated, Authentic Sources MUST notify Credential Issuers to update the validity status of any Digital Credential containing those attributes.

> Authentic Sources MUST deposit a Signal in the PDND Signal Hub :ref:`signal-hub-endpoint:Signal Collection e-Service` in the following cases:

Reviewer 2: We should add the deferred case

Reviewer 3: the section is credential revocation though

> Authentic Sources MUST deposit a Signal in the PDND Signal Hub :ref:`signal-hub-endpoint:Signal Collection e-Service` in the following cases:

Reviewer 2: ```suggestion
Authentic Sources that use Signal Hub deposit a Signal through the :ref:`signal-hub-endpoint:Signal Collection e-Service` in the following cases:
```
